product-service/src/functions/getProductById/test/handler.test.ts

```diff
@@ -0,0 +1,70 @@
+import { handler } from '../handler';
+import { getAllProducts } from '@libs/getAllProducts';
+
+const notFoundResponseExample = {
+  statusCode: 404,
+  body: 'Product not found',
+};
+
+const serverErrorResponseExample = {
+  statusCode: 500,
```

Comment:
It is better to avoid magic numbers in the code and in this case it would be great to use enum from constants
```
export enum STATUS {
  SUCCESS = 200,
  NOT_FOUND = 404,
  SERV_ERR = 500
}
```

...here and after

Reply:
Yeah, that was stupid of me to create ENUM but not to use it